Cargo.toml

```
@@ -92,6 +93,9 @@ tracing-subscriber = { version = "0.3", features = ["json", "env-filter"] }
 url = { version = "2.5", features = ["serde"] }
 uuid = "1.7"
 which = "6.0"
+openssl = { version = "0.10.68", features = ["vendored"], optional = true }
+openssl-probe = { version = "0.1.5", optional = true }
+git2 = { version = "0.19.0" , features = ["vendored-openssl"] }
```

```suggestion
git2 = { version = "0.19.0", default-features = false }
```

We don't need to use ssh or https because we don't ever interact with remote servers

---

Well, the CI/CD is working fine now with this. I don't remember why I got into this (maybe the commit?)